guardian/prosemirror-invisibles
# Add CI with github actions

## What does this change?


Currently this repo does not have any CI and therefore does not build and test upon deployment or being pushed for build.

As a result of this change issues arose surrounding the `jest-environment-jsdom` plugin and the version of `jest` compatible with it. Some dependancies have been therefore modified and updated.
## How to test


Check the `Actions` tab on in GitHub to see for successful building of a the branch.
